spec/features/user_can_view_past_orders_spec.rb

```diff
@@ -1,33 +1,63 @@
 require 'rails_helper'
 
 RSpec.feature "user can view past orders" do
-  scenario "authenticated user sees index of orders" do
-    db_repo = FactoryJordan.new
-    user = db_repo.create_users(1)[0]
+  scenario "through the order show page" do
+    user = User.create(
+      first_name: "Jordan",
+      last_name: "Lawler",
+      username: "jlawler",
+      password: "password"
+    )
+
+    ApplicationController.any_instance.stub(:current_user).and_return(user)
+
+    order = user.orders.create(
+      status: "paid",
+      total_price: 12,
+      created_at: Time.new(2016, 01, 01),
+      updated_at: Time.new(2016, 01, 02),
+      user_id: user.id
+    )
+
+    travesty = Travesty.create(title: "Environmental Disasters")
+
+    item = travesty.items.create(
+      title: "Item title",
+      description: "Item description",
+      price: 11,
+      image_url: "app/assets/images/water_contamination.jpg",
+      travesty_id: 1
+    )
+
+    order_item_1 = order.order_items.create(
+      item_id: item.id,
+      item_quantity: 4,
+      item_price: 3
+    )
 
-    visit '/'
-
-    click_link "Login"
-
-    expect(current_path).to eq login_path
-
-    fill_in "Username", with: "jlawlz"
-    fill_in "Password", with: "password"
-
-    click_button "Submit"
+    visit orders_path
 
-    user = User.find_by(username: 'jlawlz')
+    expect(page).to have_content "Past Orders"
+    expect(page).to have_link("Order ##{"%07d" % order.id.to_s}", href: order_path(order))
```

Comment:
Line is too long. [90/80]<br>Favor `format` over `String#%`.<br>Prefer single-quoted strings inside interpolations.